diff --git a/meson.build b/meson.build
index f221b0e..b8f010f 100644
--- a/meson.build
+++ b/meson.build
@@ -36,7 +36,7 @@ pkgc.generate(name: 'netorcai-client-cpp',
 )
 
 # Unit tests.
-gtest = dependency('gtest', version: '>=1.8.0', main: true, required: false)
+gtest = dependency('gtest_main', version: '>=1.8.0', required: false)
 
 test_incdir = include_directories('test', 'src')
 test_src = [
